packages/bot-web-ui/src/components/dashboard/dashboard-component/load-bot-preview/recent.tsx

```diff
@@ -29,6 +29,10 @@ const RecentComponent = ({
 }: TRecentComponent) => {
     React.useEffect(() => {
         setStrategySaveType('');
+        const getStrategies = async () => {
+            await getSavedWorkspaces().then(recent_strategies => setDashboardStrategies(recent_strategies));
```

Comment:
```suggestion
            const recent_strategies = await getSavedWorkspaces();
            setDashboardStrategies(recent_strategies);
```


Can we try doing this? @rupato-deriv

Reply:
done ✅